lib/saml_authenticator.rb

```diff
@@ -203,7 +203,7 @@ def auto_create_account(result, uid)
       user_params = {
         primary_email: UserEmail.new(email: try_email, primary: true),
         name: try_name || User.suggest_name(try_username || try_email),
-        username: UserNameSuggester.suggest(try_username || try_name || try_email || uid),
+        username: UserNameSuggester.suggest(*[try_username, try_name, try_email, uid]),
```

Comment:
I think we can skip the array here?
```suggestion
        username: UserNameSuggester.suggest(try_username, try_name, try_email, uid),
```

Reply:
Oh, exactly!